src/inputfile.rs

```diff
@@ -13,6 +13,7 @@ pub struct InputFile {
     pub accept_invalid_certs: Option<bool>,
     pub skip_tables: Option<HashSet<String>>,
     pub overrides: Option<HashMap<String, String>>,
+    pub default_features: Option<HashSet<String>>,
```

Comment:
To me, the "default" implies that this is a subset of the full set of features. Maybe just call this "features" everywhere, and call out in the command-line help that this set can be replaced (but not augmented) using the `--feature` flag?

Reply:
That makes some sense. That would leave room for something I want to add later: `--no-feature`